cpa/_tuner.py

```diff
@@ -593,17 +595,17 @@ def _trainable(
         import gc
         gc.collect()
     elif isinstance(experiment.data, (AnnData, MuData)):
-        getattr(experiment.model_cls, experiment.setup_method_name)(
-            experiment.data,
-            **experiment.setup_method_args,
-        )
-        experiment.model_cls.setup_anndata(adata, **setup_anndata_kwargs)
-
-        model = experiment.model_cls(adata, **model_args)
-        model.train(max_epochs=2000,
-                    use_gpu=True,
-                    early_stopping_patience=10,
-                    check_val_every_n_epoch=5,
+        # getattr(experiment.model_cls, experiment.setup_method_name)(
+        #     experiment.data,
+        #     # **experiment.setup_method_args,
+        # )
+        experiment.model_cls.setup_anndata(experiment.data, **setup_anndata_kwargs)
+
+        model = experiment.model_cls(experiment.data, **model_args)
+        model.train(max_epochs=train_args.pop("max_epochs",2000),
+                    use_gpu=train_args.pop("use_gpu",True),
+                    early_stopping_patience=train_args.pop("early_stopping_patience",10),
+                    check_val_every_n_epoch=train_args.pop("check_val_every_n_epoch",5),
                     plan_kwargs=plan_kwargs,
                     **train_args)
```

Comment:
Can we have this section similar to the part above it?
Eventually, something like this I think:

```suggestion
        adata = experiment.data.copy()
        if sub_sample is not None:
            sc.pp.subsample(adata, fraction=sub_sample)
        experiment.model_cls.setup_anndata(adata, **setup_anndata_kwargs)

        model = experiment.model_cls(adata, **model_args)
        model.train(plan_kwargs=plan_kwargs, **train_args)
        del adata
        import gc
        gc.collect()
```